.github/CONTRIBUTING.md

```diff
@@ -37,6 +37,10 @@ Please take a moment to review the [project readme](https://github.com/woocommer
 - Run our build process described in the document on [how to set up WooCommerce development environment](https://github.com/woocommerce/woocommerce/wiki/How-to-set-up-WooCommerce-development-environment), it will install our pre-commit hook, code sniffs, dependencies, and more.
 - Before pushing commits to GitHub, check your code against our code standards. For PHP code in the WooCommerce Core project you can do this by running `pnpm --filter=woocommerce run lint:php:changes:branch`.
 - Whenever possible, please fix pre-existing code standards errors in code that you change.
+- Please consider adding appropriate tests related to your change if applicable such as unit, API and E2E. You can check the following guides for this purpose:
+  - [Writing unit tests](https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/README.md#writing-tests).
```

Comment:
I'd rename the title of the linked section to `Guide for writing unit tests`:
- make title more clear
- align with the other linked resources below